hack/grafana-dashboards/grafana-dashboard-hive-slo.configmap.yaml

```diff
@@ -0,0 +1,1120 @@
+apiVersion: v1
+data:
+  Hive-SLO.json: |-
```

Comment:
Presently, I exported JSON from https://grafana.stage.devshift.net/goto/95cyEEgIk?orgId=1

Good question - so my idea here was to move this dashboard to both prod and stage grafana (officially even in stage under the Hive folder) and then perhaps, remove the other entries in Mike's (cc @newtonheath) or my folder, but I think it will be neat to keep those as drafts, because it is very easy to save the progress on stage grafana (unlike prod, where we don't get admin rights to our own folders). We can link that URL to share the snapshot with the person reviewing.